pkg/scheduler/scheduler.go

```diff
@@ -228,10 +218,46 @@ func (s *Scheduler) onClusterPropagationPolicyUpdate(old, cur interface{}) {
 		util.ClusterPropagationPolicyLabel: oldClusterPropagationPolicy.Name,
 	})
 
+	err := s.requeueClusterResourceBindings(selector)
+	if err != nil {
+		klog.Errorf("Failed to requeue ClusterResourceBinding, error: %v", err)
+	}
+
+	err = s.requeueResourceBindings(selector)
+	if err != nil {
+		klog.Errorf("Failed to requeue ResourceBinding, error: %v", err)
+	}
+}
+
+func (s *Scheduler) worker() {
+	for s.scheduleNext() {
+	}
+}
+
+func (s *Scheduler) requeueResourceBindings(selector labels.Selector) error {
+	referenceBindings, err := s.bindingLister.List(selector)
+	if err != nil {
+		klog.Errorf("Failed to list ResourceBinding by selector: %s, error: %v", selector.String(), err)
+		return err
+	}
+
+	for _, binding := range referenceBindings {
+		key, err := cache.MetaNamespaceKeyFunc(binding)
+		if err != nil {
+			klog.Errorf("couldn't get key for object %#v: %v", binding, err)
```

Comment:
Even though I don't know when this error would happen but log the whole object is not a good idea.
Why not using its namespace and name?

Reply:
Same as method `requeueClusterResourceBindings`.